apm-agent-core/src/main/java/co/elastic/apm/agent/bci/ElasticApmAgent.java

```diff
@@ -172,6 +181,7 @@ public void onChange(ConfigurationOption configurationOption, Object oldValue, O
                 }
             });
         }
+        tracer.start();
```

Comment:
As discussed offline, I think it's better if there's only one path to start the tracer from where it's built, rather than here